core/src/main/java/org/infinispan/distribution/group/PartitionerConsistentHash.java

```diff
@@ -77,7 +78,12 @@ public String getRoutingTableAsString() {
 
    @Override
    public boolean isKeyLocalToNode(Address nodeAddress, Object key) {
-      return ch.isSegmentLocalToNode(nodeAddress, getSegment(key));
+      // TODO Make this a method in ConsistentHash
```

Comment:
I was hesitating about what that method should be, and I thought I was short on time...